### DISCUSSION TITLE
Github not optimal, due to lengthy verification process

### DISCUSSION THREAD

**Discussion by Pinkyponki** [lsc-hubs/kenya-hub#23] [General]
The process to create the GIThub account is lengthy. The verification process took much time

**Comment by pvgenuchten** [comment]
Hi @Pinkyponki good to notice that you have succeeded. In my opinion we need to apply authentication before contributing to the hub. Github is one of the options, google/microsoft/meta or a kenyan identity provider could be an alternative. You have a favourite one?

See also #17
